R/cube-collapse-dimensions.R

```diff
@@ -0,0 +1,127 @@
+#' @rdname cube-computing
+#' @export
+setMethod("collapse.dimensions", "CrunchCube", function (x, margin=NULL) {
```

Comment:
Unless you're planning on defining different `collapse.dimensions` methods for other objects, there's no reason to define this as a method instead of a function.